### PR TITLE
Edit safe ride info

### PR DESCRIPTION
**Pull request by erichkauffman**
Change Safe Ride to Safe Guard, add info for Safe Walk, change button text

### CODE REVIEW

File: data/contact-info/1-safe-ride.yaml [hunk 1]
```diff
@@ -1,11 +1,12 @@
-title: Safe Ride
+title: Safe Guard
 phoneNumber: '5077863666'
-buttonText: Request Safe Ride
+buttonText: Call Public Safety
```
Comment (elijahverdoorn) [inline data/contact-info/1-safe-ride.yaml:3]: This is one that has been debated - while _I_ know that pubsafe = safe ride, does the majority of campus? Is it clear enough?

Comment (erichkauffman) [reply data/contact-info/1-safe-ride.yaml:3]: I think that if someone uses the app to call a safe ride, they won't be surprised when Pub Safe answers. EDIT: If the button says "Call Public Safety."

Comment (hawkrives) [reply data/contact-info/1-safe-ride.yaml:3]: I'm good with the new button text.

Comment (elijahverdoorn) [reply data/contact-info/1-safe-ride.yaml:3]: That's fine by me then, I just wanted another set of eyes.

**Review by elijahverdoorn — APPROVED** [review]

**Review by hawkrives — APPROVED** [review]

**Pull request merged**